aztec/src/main/kotlin/org/wordpress/aztec/util/Azteclog.kt

```diff
@@ -1,9 +1,42 @@
 package org.wordpress.aztec.util
 
+import org.json.JSONArray
+import org.json.JSONException
+import org.json.JSONObject
+import org.wordpress.android.util.AppLog
+import org.wordpress.aztec.AztecText
+import org.wordpress.aztec.spans.IAztecAttributedSpan
+
 class AztecLog {
     interface ExternalLogger {
         fun log(message : String)
         fun logException(tr : Throwable)
         fun logException(tr : Throwable, message : String)
     }
+
+    companion object {
+        fun logEditorContentDetails(aztecText: AztecText) {
+            try {
+                val logContentJSON = JSONObject()
+                logContentJSON.put("content", aztecText.text.toString())
+                logContentJSON.put("length", aztecText.text.length)
+                val spansJSON = JSONArray()
+                val spans = aztecText.text.getSpans(0, aztecText.text.length, IAztecAttributedSpan::class.java)
```

Comment:
There could be spans of interest that don't implement `IAztecAttributedSpan`: `UnknownHtmlSpan`, `UknownClickableSpan`, `CommentSpan`, `AztecMediaClickableSpan`, `AztecCursorSpan`, `GutenbergCommentSpan` and `WordPressCommentSpan`.

Reply:
I think the unknown spans and comment spans could be useful, since they could contain HTML that affects parsing.

Reply:
Right, I've changed the code so all types of spans are logged in case of crash. Better to log everything.

Reply:
I don't see the change. Maybe you forgot to push your commit?

Reply:
🤦‍♂️